client/src/pages/Projects.js

```diff
@@ -0,0 +1,59 @@
+import { Template } from "../components/Template";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import ThemeProvider from "react-bootstrap/ThemeProvider";
+
+export const Projects = () => {
+	return (
+		<Template>
+			<ThemeProvider
+				breakpoints={["xxxl", "xxl", "xl", "lg", "md", "sm", "xs", "xxs"]}
+				minBreakpoint="xxs"
+			>
```

Comment:
You can use this to wrap App.js

Reply:
Then you can use the breakpoints like in the documentation https://react-bootstrap.github.io/layout/grid/#responsive-grids

Reply:
This would be an example of a column that takes full width on small screens `xs` and 2/3 width from medium screens upwards `md`, the width is divided into 12
```html
 <Col xs={12} md={8}>
          xs=12 md=8
        </Col>
```